bin.js

```diff
@@ -36,6 +36,8 @@ args
   .option(['e', 'errorProps'], 'Comma separated list of properties on error objects to show (`*` for all properties) (defaults to ``)')
   .option(['l', 'levelFirst'], 'Display the log level as the first output field')
   .option(['L', 'minimumLevel'], 'Hide messages below the specified log level')
+  .option(['X', 'customLevels'], 'Override default levels (`-cl err:99,info:1`)')
+  .option(['Y', 'customColors'], 'Override default colors using names from https://www.npmjs.com/package/colorette (`-cc err:red,info:blue`)')
```

Comment:
@mcollina `-X` and `-Y` are just random 1 char option names... do you have any suggestions to how we should call these options?

I've tried to remove the 1 char name, but it defaults the name to `-C` on both.

Reply:
Let's just avoid/remove the one-char option.

Reply:
I'll try that again and post here the help result 👍

Reply:
Removing the one-char option is not a solution, `customColors` and `customLevels` now have a default `-C` which is also used for `--config`. See example below:
```sh
  Usage: pino pretty [options] [command]
  
  Commands:
    help     Display help
    version  Display version
  
  Options:
    -c, --colorize              Force adding color sequences to the output
    -C, --config                specify a path to a json file containing the pino-pretty options
    -f, --crlf                  Append CRLF instead of LF to formatted lines
    -C, --customColors          Override default colors using names from https://www.npmjs.com/package/colorette (`-cc err:red,info:blue`)
    -C, --customLevels          Override default levels (`-cl err:99,info:1`)
    -k, --errorLikeObjectKeys   Define which keys contain error objects (`-k err,error`) (defaults to `err,error`)
    -e, --errorProps            Comma separated list of properties on error objects to show (`*` for all properties) (defaults to ``)
    -h, --help                  Output usage information
    -H, --hideObject            Hide objects from output (but not error object)
    -i, --ignore                Ignore one or several keys: (`-i time,hostname`)
    -l, --levelFirst            Display the log level as the first output field
    -L, --levelKey [value]      Detect the log level under the specified key (defaults to "level")
    -b, --levelLabel [value]    Output the log level using the specified label (defaults to "levelLabel")
    -o, --messageFormat         Format output of message
    -m, --messageKey [value]    Highlight the message under the specified key (defaults to "msg")
    -L, --minimumLevel          Hide messages below the specified log level
    -S, --singleLine            Print all non-error objects on a single line
    -a, --timestampKey [value]  Display the timestamp from the specified key (defaults to "time")
    -t, --translateTime         Display epoch timestamps as UTC ISO format or according to an optional format string (default ISO 8601)
    -v, --version               Output the version number
  
  Examples:
...
```

Any chance we can come up with a one-char for these two options?

Reply:
See https://github.com/leo/args/issues/145